### PR TITLE
aaryaman 😺

### PR DESCRIPTION
**Pull request by aaryamvn**

### CODE REVIEW

File: utils/Consts.ts [hunk 1]
```diff
@@ -53,4 +53,10 @@ after Capybara api: :ChocolaPOGGERS:`,
     pfp: "/img/ari.png",
     worksAt: "Buildergroop Corporation",
   },
+  {
+    message: "man capybara api be bussin yo this mf gon replace your enterprise-grade fancy ass graphql apis. p.s shoutout to my co-founder ari this dude gets it too yo capybara shit eating ritual mad healthy might partake one day.",
```
Comment (looskie) [inline utils/Consts.ts:57]: can you fix this indentation

**Review by looskie — CHANGES_REQUESTED** [review]

File: utils/Consts.ts (hunk 1 above)
Comment (aaryamvn) [reply utils/Consts.ts:57]: i can not

Comment (aaryamvn) [reply utils/Consts.ts:57]: I hope you're satisfied with my latest commit, "Looskie".

Comment (looskie) [reply utils/Consts.ts:57]: i am thx

**Review by looskie — APPROVED** [review]
lgtm, i will need to get my goons to also review this PR

**Comment by robjmorrissey** [comment]
please

**Review by Phineas — CHANGES_REQUESTED** [review]
This is so unprofessional. Not what I'd expect to see in a capybara-web PR.

**Review by Phineas — APPROVED** [review]
sike WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW

**Review by Phineas — CHANGES_REQUESTED** [review]
actually wait - still reviewing the code
give me a couple months

**Comment by looskie** [comment]
seems like the goons like the PR, thank you for contributing @aaryamvn

**Comment by looskie** [comment]
@aaryamvn After further examination, Phineas is still reviewing, please reopen this PR.

**Review by maxichrome — COMMENTED** [review]
reasonably vibes. fit for merge. easily.

**Comment by looskie** [comment]
Closed with PR #3

**Comment by aaryamvn** [comment]
I am highly grateful for all of your goons (namely @Phineas, @maxichrome, and @robjmorrissey) given that they took time out of their incredibly busy days to review my PR. The fact that all of you spared seconds, minutes, or in some cases - literal months (ly @Phineas) to review my contribution to a piece of software that I personally believe, has the power to change the way we interact with API's for good, makes me incredibly content and my faith in humanity has been restored.

I appreciate all of you so much. Thank you for making this possible.

**Pull request merged**